vitest.config.ts

```diff
@@ -1,9 +1,8 @@
 import { defineConfig } from 'vitest/config';
-import tsconfigPaths from 'vite-tsconfig-paths';
 
 export default defineConfig({
   test: {
     environment: 'jsdom',
   },
-  plugins: [tsconfigPaths()],
+  plugins: [],
```

Comment:
hmm lets see what we needed this for, might be worth keeping this plugin

Reply:
Was added as part of the #649 PR, happy to keep it.

I don't see any paths defined in tsconfig.json for which we'd need this vite path mapping plugin though